### PR TITLE
[NUI] Remove [Obsolete()] attribute

### PR DESCRIPTION
**Pull request by dongsug-song**
### Description of Change ###
[NUI] Remove [Obsolete()] attribute

### API Changes ###
https://code.sec.samsung.net/jira/browse/TCSACR-391

### CODE REVIEW

**Comment by m-macias** [comment]
If I recall correctly, you should create a similar pull request for the master branch, because it is a current development branch and those changes should be included in the API9. Only one ACR is required for both branches.

**Comment by dongsug-song** [comment]
> If I recall correctly, you should create a similar pull request for the master branch, because it is a current development branch and those changes should be included in the API9. Only one ACR is required for both branches.

@m-macias 
Hello, here I made a PR for master : https://github.com/Samsung/TizenFX/pull/2248
actually this is in DevelNUI, but all patches in DevelNUI are integrated and merged into master once a every week.
thank you.

**Review by huiyueun — APPROVED** [review]

**Comment by dongsug-song** [comment]
@m-macias 
Hello, 
TCSACR-391 is accepted, so can I merge this?
please confirm it.
thank you.

**Comment by dongsug-song** [comment]
https://code.sec.samsung.net/jira/browse/TCSACR-391 is accepted, so I merge this now.

File: src/Tizen.NUI/src/public/Color.cs [hunk 1]
```diff
@@ -207,8 +217,18 @@ public float R
         /// <summary>
         /// The green component.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Color(...) constructor.
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Color.cs:221]:
```suggestion
        /// The setter is deprecated in API8 and will be removed in API10. Use the new Color(...) constructor.
```

File: src/Tizen.NUI/src/public/Color.cs [hunk 2]
```diff
@@ -184,8 +184,18 @@ internal Color(ColorChangedCallback cb, Color other) : this(cb, other.R, other.G
         /// <summary>
         /// The red component.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Color(...) constructor.
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Color.cs:188]:
```suggestion
        /// The setter is deprecated in API8 and will be removed in API10. Use the new Color(...) constructor.
```

File: src/Tizen.NUI/src/public/Color.cs [hunk 3]
```diff
@@ -207,8 +217,18 @@ public float R
         /// <summary>
         /// The green component.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Color(...) constructor.
+        /// </remarks>
+        /// <code>
+        /// // DO NOT use like the followings!
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Color.cs:224]:
```suggestion
        /// // DO NOT use as follows:
```

File: src/Tizen.NUI/src/public/Color.cs [hunk 4]
```diff
@@ -230,8 +250,18 @@ public float G
         /// <summary>
         /// The blue component.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Color(...) constructor.
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Color.cs:254]:
```suggestion
        /// The setter is deprecated in API8 and will be removed in API10. Use the new Color(...) constructor.
```

File: src/Tizen.NUI/src/public/Color.cs [hunk 5]
```diff
@@ -230,8 +250,18 @@ public float G
...
+        /// </remarks>
+        /// <code>
+        /// // DO NOT use like the followings!
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Color.cs:257]:
```suggestion
        /// // DO NOT use as follows:
```

File: src/Tizen.NUI/src/public/Color.cs [hunk 6]
```diff
@@ -253,8 +283,18 @@ public float B
         /// <summary>
         /// The alpha component.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Color(...) constructor.
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Color.cs:287]: Please see above.

File: src/Tizen.NUI/src/public/Color.cs [hunk 7]
```diff
@@ -253,8 +283,18 @@ public float B
...
+        /// </remarks>
+        /// <code>
+        /// // DO NOT use like the followings!
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Color.cs:290]: Please see above.

File: src/Tizen.NUI/src/public/Extents.cs [hunk 8]
```diff
@@ -122,8 +122,18 @@ public void CopyFrom(Extents that)
         /// <summary>
         /// The Start extent.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Extents(...) constructor.
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Extents.cs:126]: Please see above.

File: src/Tizen.NUI/src/public/Extents.cs [hunk 9]
```diff
@@ -122,8 +122,18 @@ public void CopyFrom(Extents that)
...
+        /// </remarks>
+        /// <code>
+        /// // DO NOT use like the followings!
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Extents.cs:129]: Please see above.

File: src/Tizen.NUI/src/public/Extents.cs [hunk 10]
```diff
@@ -145,8 +155,18 @@ public ushort Start
         /// <summary>
         /// The End extend.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Extents(...) constructor.
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Extents.cs:159]: Please see above.

File: src/Tizen.NUI/src/public/Extents.cs [hunk 11]
```diff
@@ -145,8 +155,18 @@ public ushort Start
...
+        /// </remarks>
+        /// <code>
+        /// // DO NOT use like the followings!
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Extents.cs:162]: Please see above.

File: src/Tizen.NUI/src/public/Extents.cs [hunk 12]
```diff
@@ -169,8 +189,18 @@ public ushort End
         /// <summary>
         /// The Top extend.
         /// </summary>
+        /// <remarks>
+        /// The setter is deprecated in API8 and will be removed in API10. Please use new Extents(...) constructor.
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Extents.cs:193]: Please see above.

File: src/Tizen.NUI/src/public/Extents.cs [hunk 13]
```diff
@@ -169,8 +189,18 @@ public ushort End
...
+        /// </remarks>
+        /// <code>
+        /// // DO NOT use like the followings!
```
Comment (journey2w) [inline src/Tizen.NUI/src/public/Extents.cs:196]: Please see above.

**Review by journey2w — COMMENTED** [review]
Please update all same descriptions.

File: src/Tizen.NUI/src/public/Color.cs (hunk 1 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Color.cs:221]: thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118

File: src/Tizen.NUI/src/public/Color.cs (hunk 2 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Color.cs:188]: thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118

File: src/Tizen.NUI/src/public/Color.cs (hunk 3 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Color.cs:224]: thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118

File: src/Tizen.NUI/src/public/Color.cs (hunk 4 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Color.cs:254]: thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118

File: src/Tizen.NUI/src/public/Color.cs (hunk 5 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Color.cs:257]: thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118

File: src/Tizen.NUI/src/public/Extents.cs (hunk 9 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Extents.cs:129]: thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118

File: src/Tizen.NUI/src/public/Extents.cs (hunk 10 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Extents.cs:159]: thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118

File: src/Tizen.NUI/src/public/Extents.cs (hunk 11 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Extents.cs:162]: thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118

File: src/Tizen.NUI/src/public/Extents.cs (hunk 12 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Extents.cs:193]: thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118

File: src/Tizen.NUI/src/public/Extents.cs (hunk 13 above)
Comment (dongsug-song) [reply src/Tizen.NUI/src/public/Extents.cs:196]: thank you for review.
I have fixed by https://github.com/Samsung/TizenFX/pull/4118

**Pull request merged**